### ISSUE TITLE
AbstractProblemHoverTest fails on Eclipse CI

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-eclipse#662]
AbstractProblemHoverTest fails on Eclipse CI, maybe an order problem

```
Fehlermeldung

expected:<...pse.xtext.ui.editor.[info]> but was:<...pse.xtext.ui.editor.[error]>

Stacktrace

org.junit.ComparisonFailure: expected:<...pse.xtext.ui.editor.[info]> but was:<...pse.xtext.ui.editor.[error]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at org.eclipse.xtext.ui.tests.editor.hover.AbstractProblemHoverTest.testMultipleAnnotationTypes(AbstractProblemHoverTest.java:110)
```

**Comment by cdietrich** [comment]
@meysholdt i am not sure why this test does not pick up your changes on the eclipse ci server

**Comment by spoenemann** [comment]
The test fails for the unsorted list, which takes the order of annotations exactly as given by the annotation model. If that is not stable across systems, we should remove that part of the test. The following one, which tests the `sortBySeverity` method, is more important anyway.

**Comment by cdietrich** [comment]
Fixed in 2.14
